src/dswx_sar/pre_processing.py

```diff
@@ -240,15 +290,18 @@ def run(cfg):
     ancillary_reloc = AncillaryRelocation(ref_filename, scratch_dir)
 
     # Check if the interpolated water body file exists
-    wbd_interpolated_path = Path(os.path.join(scratch_dir,
-                                              'interpolated_wbd.tif'))
+    wbd_interpolated_path = Path(
+        os.path.join(scratch_dir, 'interpolated_wbd.tif'))
```

Comment:
```suggestion
    wbd_interpolated_path = Path(scratch_dir) / 'interpolated_wbd.tif'
```
Same for `dem_interpolated_path` and `landcover_interpolated_path`